go.mod

```diff
@@ -41,33 +41,40 @@ replace (
 )
 
 replace (
+	// github.com/firecracker-microvm/firecracker-containerd => github.com/ease-lab/firecracker-containerd v0.0.0-20210618165033-6af02db30bc4
+	github.com/containerd/containerd => github.com/ease-lab/containerd v1.5.5-ids // TODO: change to vhive
 	github.com/ease-lab/vhive/examples/protobuf/helloworld => ./examples/protobuf/helloworld
-	github.com/firecracker-microvm/firecracker-containerd => github.com/ease-lab/firecracker-containerd v0.0.0-20210618165033-6af02db30bc4
+	github.com/firecracker-microvm/firecracker-containerd => github.com/amohoste/firecracker-containerd v1.0.0-enhanced-snap // TODO: change to vhive
 )
 
 require (
+	github.com/antchfx/xpath v1.2.0 // indirect
 	github.com/blend/go-sdk v1.20211025.3 // indirect
-	github.com/containerd/containerd v1.5.2
+	github.com/containerd/containerd v1.5.8
```

Comment:
Recently containers got upgraded to 1.6 in firecracker-containerd. Can you upgrade in vHive as well, rebasing firecracker-containerd again and upgrading it here in go.mod files